obal/data/playbooks/copr-project/copr-project.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Configure Copr projects
+  hosts:
+    - copr_projects
+    - packages
```

Comment:
Hoping to avoid having to set packages here via https://github.com/theforeman/obsah/pull/25